.travis.yml

```diff
@@ -1,124 +1,169 @@
+    - name: "build release on lowest supported toolchain"
+      rust: 1.41.0
+      env: TASK=release
+
+
+    # MANDATORY TESTING ON STABLE
+    - name: "build using stable toolchain"
+      rust: stable
+      env: TASK=build
+    - name: "build without defaults using stable toolchain"
+      rust: stable
+      env: TASK=build-no-default
+    - name: "build Rust docs using stable toolchain"
+      rust: stable
+      env: TASK=docs-travis
+    - name: "run Rust unit tests using stable toolchain"
+      rust: stable
+      env: TASK=test
+    - name: "run Rust destructive unit tests using stable toolchain"
+      rust: stable
+      script:
+        - >
+          sudo
+          PATH=${TRAVIS_HOME}/.cargo/bin:$PATH RUST_LOG=libstratis=info
+          make
+          -f
+          Makefile
+          test-travis
+    - name: "build release using stable toolchain"
+      rust: stable
+      env: TASK=release
+
+
+    # MANDATORY PYTHON CHECKS ON RECOMMENDED DEVELOPMENT INTERPRETER
+    - name: "lint Python code on recommended development interpreter"
+      language: python
+      python: "3.7.6"
+      install:
+        - >
+          pip
+          install
+          pylint==2.3.1
+          dbus-client-gen==0.4
+          dbus-python-client-gen==0.7
+          psutil==5.4.3
+          pyudev==0.22.0
+      script:
+        - >
+          (cd scripts; make -f Makefile lint);
+          (cd tests/client-dbus; PYTHONPATH=./src make -f Makefile lint)
+    - name: "format Python code on recommended development interpreter"
+      language: python
+      python: "3.7.6"
+      install: pip install black==19.3b0 isort==4.3.4
+      script:
+        - >
+          (cd scripts; make fmt-travis);
+          (cd tests/client-dbus; make fmt-travis)
+
+    # MANDATORY PYTHON CHECKS ON LOWEST SUPPORTED INTERPRETER
+    - name: "lint Python code on lowest supported interpreter"
+      language: python
+      python: "3.6.8"
+      install:
+        - >
+          pip
+          install
+          pylint==2.3.1
+          dbus-client-gen==0.4
+          dbus-python-client-gen==0.7
+          psutil==5.4.3
+          pyudev==0.22.0
+      script:
+        - >
+          (cd scripts; make -f Makefile lint);
+          (cd tests/client-dbus; PYTHONPATH=./src make -f Makefile lint)
+
+    # INTERMITTENTLY ALLOWED FAILURES
+    # Allowed if a failure occurs after a new Rust release until the
+    # failure is addressed.
+    - name: "lint Rust source using beta toolchain"
+      rust: beta
+      before_script:
+        - rustup component add clippy
+      env: TASK=clippy
+
+
+    # ALLOWED FAILURES
+    # Run audit on Rust stable.  Make it an allowed failure, because:
+    # * It takes 9 minutes, the longest of any task.
+    # * It should be an advisory, and should not gate our development.
+    - name: "run Rust audit task using stable toolchain"
+      rust: stable
+      env: TASK=audit
+
+    - name: "run yamllint 1.20.0 on .travis.yml"
```

Comment:
Please add a comment here to separate it out from ```ALLOWED FAILURES``` and put this task  above the INTERMITTENTLY ALLOWED  and the ```ALLOWED FAILURES```.

Call the new section something like ```VERIFICATION OF TEST INFRASTRUCTURE```.

Reply:
Resolved.